sdks/java/core/src/main/java/org/apache/beam/sdk/util/ApiSurface.java

```diff
@@ -825,9 +825,9 @@ private boolean exposed(int modifiers) {
    */
   public static ApiSurface getSdkApiSurface(final ClassLoader classLoader) throws IOException {
     return ApiSurface.ofPackage("org.apache.beam", classLoader)
-        .pruningPattern("org[.]apache[.]beam[.].*Test")
-        // Exposes Guava, but not intended for users
-        .pruningClassName("org.apache.beam.sdk.util.common.ReflectHelpers")
-        .pruningPrefix("java");
+                     .pruningPattern("org[.]apache[.]beam[.].*Test")
```

Comment:
I would prefer to not change the indentation scheme "incidentally", but up to you.